src/Ionide.ProjInfo.ProjectSystem/WorkspacePeek.fs

```diff
@@ -72,7 +74,21 @@ module WorkspacePeek =
                 | UsefulFile.Slnf ->
                     match InspectSln.tryParseSln f.FullName with
                     | Ok (p, d) -> Some(Choice1Of3(p, d))
-                    | _ -> None
+                    | Error e ->
+                        let addInfo l =
+                            match e with
+                            | :? Ionide.ProjInfo.Sln.Exceptions.InvalidProjectFileException as ipfe -> Log.addContextDestructured "data" ipfe l
```

Comment:
The reason I'm adding the exception as `addContextDestructured` is the Properties on this exception weren't getting logged by Serilog and I didn't investigate deep in how to configure Serilog to do it.